### DISCUSSION TITLE
date comparison in jslt

### DISCUSSION THREAD

**Discussion by kuanghsuan** [schibsted/jslt#258] [General]
Is there any way for us to get the current date and compare with "2021-01-01" to see which one is earlier?

**Comment by larsga** [comment]
In that date format comparing dates as strings works the  same as comparing dates directly, so the comparison is easy.

[now](https://github.com/schibsted/jslt/blob/master/functions.md#now---double) gets you the current date and time.

[format-time](https://github.com/schibsted/jslt/blob/master/functions.md#format-timetimestamp-format-timezone---string) will let you convert the date+time to the date format you need.
